### PR TITLE
adds relevant gems

### PR DESCRIPTION
**Pull request by Dpalazzari**
Added some gems to Gemfile

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -28,20 +29,33 @@ gem 'jbuilder', '~> 2.5'
...
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platform: :mri
+  gem 'pry'
+  gem 'rspec-rails', '~> 3.5'
+  gem 'capybara'
+  gem 'launchy'
+  gem 'factory_girl_rails'
+  gem 'database_cleaner'
+  gem 'faker'
```
Comment (njgheorghita) [inline Gemfile:48]: great gem choice!

**Review by njgheorghita — COMMENTED** [review]
nice gem selection

File: Gemfile (hunk 1 above)
Comment (Laszlo-JFLMTCO) [reply Gemfile:48]: Looks good! 
Thank you @Dpalazzari  for setting up the gems!

Comment (drod1000) [reply Gemfile:48]: Looks good!

**Pull request merged**